workflows/toast_sim_ground.py

```diff
@@ -250,6 +255,11 @@ def reduce_data(job, args, data):
     ops.common_mode_filter.apply(data)
     log.info_rank("Finished common-mode-filtering in", comm=world_comm, timer=timer)
 
+    # Collect signal statistics after filterting
+
+    ops.filtered_statistics.output_dir = args.out_dir
```

Comment:
This is the same output directory used by the previous dump- will it overwrite those files?  Maybe dump to sub-directories or support a "prefix" trait in the `Statistics` operator to make the file names unique?

Reply:
I'm using the unique operator name as the prefix. No overwriting should occur.